Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementTextInput.cshtml

```diff
@@ -5,15 +5,15 @@
 }
 
 <div class="govuk-form-group">
-    @if (!string.IsNullOrWhiteSpace(Model.Caption))
-    {
-        <span class="govuk-caption-l">@Model.Caption</span>
-    }
-
+   
     @if (!string.IsNullOrWhiteSpace(Model.Heading))
     {
         <label class="govuk-label govuk-label--l" for="declarationInput">@Model.Heading</label>
```

Comment:
Same as above - the label is not associated with anything

Reply:
@andymantell well spotted Andy, I have removed it.